packages/material-ui/src/Slider/Slider.js

```diff
@@ -352,6 +385,18 @@ const Slider = React.forwardRef(function Slider(inputProps, ref) {
         MarkLabel: SliderMarkLabel,
         ...components,
       }}
+      componentsProps={{
```

Comment:
Maybe it's better to use just `deepmerge`

Reply:
It could be simpler, what to do with `isHostComponent`?